### ISSUE TITLE
internal TF: grant Packit admins ability to reschedule internal TF runs

### ISSUE THREAD

**Issue by TomasTomecek** [packit/packit-service#1362]
relevant code: https://github.com/packit/packit-service/blob/main/packit_service/worker/handlers/testing_farm.py#L99

```
16:26 <mvadkert> testing-farm:epel-7-x86_64 — thrix can't run tests internally
16:26 ⚡ mvadkert is sad :D
16:26 <mvadkert> can sombody give me that powa?
```

Let's make @thrix happy, not sad.

**Comment by thrix** [comment]
thanks! not a priority, but more power feels good, I can fix issues for users what makes them happy.

**Comment by TomasTomecek** [comment]
the same applies for us, we also cannot reschedule them

**Comment by mfocko** [comment]
Should be fixed in eb6f33420f43b1bb5b41cb72f18e9aa06235fdbc
